submission/persistence-filesystem/src/main/kotlin/ac/uk/ebi/biostd/persistence/filesystem/fire/FireFtpService.kt

```diff
@@ -23,36 +24,16 @@ class FireFtpService(
         logger.info { "$accNo $owner Finished publishing files of submission $accNo over FIRE" }
     }
 
-    override fun unpublishSubmissionFiles(accNo: String, owner: String, relPath: String) {
-        logger.info { "$accNo $owner Un-publishing files of submission $accNo over FIRE" }
-
-        cleanFtpFolder(accNo)
-
-        logger.info { "$accNo $owner Finished un-publishing files of submission $accNo over FIRE" }
-    }
-
     override fun generateFtpLinks(accNo: String) {
         val sub = queryService.getExtByAccNo(accNo, includeFileListFiles = true)
 
         logger.info { "${sub.accNo} ${sub.owner} Started processing FTP links for submission $accNo over FIRE" }
-
-        serializationService.forEachSubmissionFile(sub) { if (it is FireFile) publishFile(it.fireId) }
+        serializationService.forEachFile(sub) { file, idx -> if (file is FireFile) publishFile(sub, file.fireId, idx) }
         logger.info { "${sub.accNo} ${sub.owner} Finished processing FTP links for submission $accNo over FIRE" }
     }
 
-    private fun cleanFtpFolder(accNo: String) {
-        fireClient
-            .findByAccNoAndPublished(accNo, true)
-            .forEach { unPublishFile(it.fireOid) }
-    }
-
-    private fun publishFile(fireId: String) {
+    private fun publishFile(sub: ExtSubmission, fireId: String, index: Int) {
+        logger.debug { "${sub.accNo}, ${sub.version} publishing file $index, fireId='$fireId'" }
```

Comment:
I think the agreed format for the logs was accNo/owner. That is if we're still using the kibana thing